config/initializers/limits.rb

```diff
@@ -2,3 +2,7 @@
 def secrets_version_limit
   ( ENV['SECRETS_VERSION_LIMIT'] || 20 ).to_i
 end
+
+# Max size of policies that can be loaded (or data sent to POST endpoints in
+# general)
+Rack::Utils.key_space_limit = 1048576 * 10 # 10 MiB
```

Comment:
It wasn't clear to me why setting an attribute named `key_space_limit` would change the maximum policy size. A little more explanation here, describing how they're related, would be good.